dotnet/samples/KernelSyntaxExamples/README.md

```diff
@@ -161,10 +161,17 @@ KeyVault__TenantId
 HuggingFace__ApiKey
 HuggingFace__ModelId
 
-# Gemini
-Gemini__ApiKey
-Gemini__ModelId
-Gemini__EmbeddingModelId
+# GoogleAI
+GoogleAI__ApiKey
+GoogleAI__EmbeddingModelId
+GoogleAI__Gemini__ModelId
+
+# VertexAI
+GoogleAI__ApiKey
+GoogleAI__EmbeddingModelId
+GoogleAI__Location
+GoogleAI__ProjectId
+GoogleAI__Gemini__ModelId
```

Comment:
Should these use a seperate prefix?

E.g. if I want to run Gemini and VertexAI together with a different API key?

Reply:
same Gemini_ModelId It's better? It can be common for GoogleAI and VertexAI.
or shortly GoogleAI_ModelId similar to openaai and azure openai.

Yeah, googleai and vertexai have others keys, but same gemini models.

EmbeddingModelId is other for GoogleAI and other for VertexAI.